cobbler/actions/reposync.py

```diff
@@ -207,11 +202,11 @@ def sync(self, repo):
         elif repo.breed == RepoBreeds.WGET:
             self.wget_sync(repo)
         else:
-            utils.die("unable to sync repo (%s), unknown or unsupported repo type (%s)" % (repo.name, repo.breed.value))
+            raise CX("unable to sync repo (%s), unknown or unsupported repo type (%s)" % (repo.name, repo.breed.value))
 
     # ====================================================================================
 
-    def librepo_getinfo(self, dirname):
+    def librepo_getinfo(self, dirname: str) -> dict:
```

Comment:
Since you digged into this function, I would love if it is possible to add a docstring for this function.